sources/platform/actors/development/quick_start/start_web_ide.md

```diff
@@ -5,91 +5,143 @@ description: Create your first Actor using the web IDE in Apify Console.
 slug: /actors/development/quick-start/web-ide
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Development in web IDE
 
 **Create your first Actor using the web IDE in Apify Console.**
 
 ---
 
-## 1. Create the Actor
+## Create the Actor
+
+:::note Prerequisites
+
+To use Web IDE, you will need an Apify account. You can [sign-up for a free account](https://console.apify.com/sign-up) on the Apify website.
 
-> You will need an Apify account to complete this tutorial. If you don't have one, you can [sign-up for a free account](https://console.apify.com/sign-up) on the Apify website.
+:::
 
-After you sign in to [Apify Console](https://console.apify.com), navigate to the [Actors](https://console.apify.com/actors) section on your left panel. Then, click the **Create new** button at the top right corner of the page.
+After you sign in to [Apify Console](https://console.apify.com), navigate to the [**Actors**](https://console.apify.com/actors) section. Then, click the **Develop new** button at the top right corner of the page.
 
 ![Create Actor](./images/actor-create-button.png)
 
-You will be redirected to a page containing various Actor development templates for popular languages such as JavaScript, TypeScript, and Python. You can choose the template that best suits your technology stack. For demonstration purposes, let's choose "PuppeteerCrawler in JavaScript".
+You will be redirected to a page containing various Actor development templates for popular languages such as `JavaScript`, `TypeScript`, and `Python`. These templates provide boilerplate code and a preconfigured environment tailored to specific use cases. You can choose the template that best suits your technology stack. For demonstration purposes, let's choose **Crawlee + Puppeteer + Chrome**.
 
 ![Templates](./images/actor-create-templates.png)
 
-You will be prompted to give your Actor a name and then redirected to your Actor configuration page.
+A prompt will ask you to give your Actor a name and then redirected to your Actor configuration page.
```

Comment:
There is no such thing anymore. You'll be redirected straight away to the Actor page (the Actor is name automatically and you can rename it on the Actor)

Reply:
removed the mention about prompt, and changed so it mentions that the actor will be automatically named and user will be redirected to its page